src/content/components/TimelineCanvas.css

```diff
@@ -0,0 +1,9 @@
+.timelineCanvas {
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.timelineCanvas.hover {
+  cursor: default;
```

Comment:
The behavior here is to show the `grab` cursor on the Timeline when not hovering over a stack frame, then when you hover over a stack frame it changes to the `default` cursor.